recipe/meta.yaml

```
@@ -1,28 +1,32 @@
 {% set name = "xalan-c" %}
-{% set version = "1.11" %}
+{% set version = "1.12" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  # xalan lib uses <major>_<minor> in links intead of <major>.<minor>
-  url: https://github.com/apache/xalan-c/archive/Xalan-C_{{ version|replace(".","_") }}_0.tar.gz
-  sha256: edfaee4562d9dec4d0a971e06cc6355f8c822ce49f18beba095f8a4dc93b4285
+  # xalan lib uses <major>_<minor> in links instead of <major>.<minor>
+  url: https://github.com/apache/xalan-c/archive/refs/tags/Xalan-C_{{ version|replace(".","_") }}_0.zip
+  sha256: 50bb8107c41c9e65410371e1272c66cb49531b9a3b2a55ccb62ad57dd3a35a4d
 
 build:
   skip: True  # [win]
   number: 0
 
 requirements:
   build:
+    - cmake >=3.15
     - make
```

This simply will give you better error messages.
```suggestion
    - ninja
```